src/main/java/org/swisspush/reststorage/RedisStorage.java

```diff
@@ -700,17 +724,33 @@ public ByteArrayWriteStream exceptionHandler(Handler<Throwable> handler) {
         }
 
         @Override
-        public WriteStream<Buffer> write(Buffer data) {
+        public Future<Void> write(Buffer data) {
             try {
                 bos.write(data.getBytes());
+                return Future.succeededFuture();
             } catch (IOException e) {
-                throw new RuntimeException(e);
+                return Future.failedFuture(e.getMessage());
             }
-            return this;
+
+        }
+
+        @Override
+        public void write(Buffer data, Handler<AsyncResult<Void>> handler) {
+

```

Comment:
Is "*do nothing*" the right thing to do here?

Reply:
I call another write